### PR TITLE
[p5js - HI] Translation of 3D pages to hindi 

### PR DESCRIPTION
**Pull request by jaiakash**
Hi, i have completed the 3D pages translation in hindi. @TanviKumar @Divyansh013 can you please review this PR and let me know the changes if any.

Thanks !

### CODE REVIEW

File: src/content/reference/hi/p5.Camera/camera.mdx [hunk 1]
```diff
@@ -1,56 +1,12 @@
 ---
 title: camera
-module: 3D
-submodule: Camera
+module: 3डी
```
Comment (TanviKumar) [inline src/content/reference/hi/p5.Camera/camera.mdx:3]: Leave 3D as 3D?

File: src/content/reference/hi/p5.Camera/camera.mdx [hunk 2]
```diff
@@ -187,47 +143,47 @@ class: p5.Camera
...
   - name: z
     description: |
-      <p>z-coordinate of the camera. Defaults to 0.</p>
+      <p>कैमरे का z-समन्वय। डिफ़ॉल्ट 0.</p>
     type: Number
     optional: true
   - name: centerX
     description: |
-      <p>x-coordinate of the point the camera faces. Defaults to 0.</p>
+      <p>कैमरे के सम्मुख बिंदु का x-निर्देशांक। डिफ़ॉल्ट 0.</p>
```
Comment (TanviKumar) [inline src/content/reference/hi/p5.Camera/camera.mdx:161]: Use consistent word for coordinate as discussed

File: src/content/reference/hi/p5.Camera/centerY.mdx [hunk 3]
```diff
@@ -1,15 +1,12 @@
 ---
 title: centerY
-module: 3D
-submodule: Camera
+module: 3डी
```
Comment (TanviKumar) [inline src/content/reference/hi/p5.Camera/centerY.mdx:3]: 3D across the files

File: src/content/reference/hi/p5.Camera/lookAt.mdx [hunk 4]
```diff
@@ -89,18 +79,15 @@ class: p5.Camera
...
     description: >
-      <p>x-coordinate of the position where the camera should look in "world"
-      space.</p>
+      <p>x-उस स्थिति का समन्वय जहां कैमरे को "विश्व" स्थान में देखना चाहिए।</p>
     type: Number
   - name: 'y'
     description: >
-      <p>y-coordinate of the position where the camera should look in "world"
-      space.</p>
+      <p>y-उस स्थिति का समन्वय जहां कैमरे को "विश्व" स्थान में देखना चाहिए।</p>
```
Comment (TanviKumar) [inline src/content/reference/hi/p5.Camera/lookAt.mdx:86]: Consistent coordinate use across files

File: src/content/reference/hi/p5.Shader/copyToContext.mdx [hunk 5]
```diff
@@ -1,60 +1,17 @@
 ---
 title: copyToContext
-module: 3D
-submodule: Material
+module: 3डी
+submodule: भौतिक
```
Comment (TanviKumar) [inline src/content/reference/hi/p5.Shader/copyToContext.mdx:4]: Can we use a simpler translation for material?

**Review by TanviKumar — CHANGES_REQUESTED** [review]

File: src/content/reference/hi/p5.Camera/camera.mdx (hunk 1 above)
Comment (jaiakash) [reply src/content/reference/hi/p5.Camera/camera.mdx:3]: Sure, Updated

File: src/content/reference/hi/p5.Camera/centerY.mdx (hunk 3 above)
Comment (jaiakash) [reply src/content/reference/hi/p5.Camera/centerY.mdx]: Done

File: src/content/reference/hi/p5.Shader/copyToContext.mdx (hunk 5 above)
Comment (jaiakash) [reply src/content/reference/hi/p5.Shader/copyToContext.mdx:4]: For Material, i have 3 suggestions - 
- सामग्री
- पदार्थ
- द्रव्यमान

In my opinion, द्रव्यमान seems most suited for the given context. @TanviKumar ?

File: src/content/reference/hi/p5/orbitControl.mdx [hunk 6]
```diff
@@ -176,24 +139,22 @@ class: p5
...
   - name: sensitivityY
     description: |
-      <p>sensitivity to movement along the y-axis. Defaults to 1.</p>
+      <p>y-अक्ष के साथ गति के प्रति संवेदनशीलता। डिफ़ॉल्ट 1</p> है
     type: Number
     optional: true
   - name: sensitivityZ
     description: |
-      <p>sensitivity to movement along the z-axis. Defaults to 1.</p>
+      <p>z-अक्ष के साथ गति के प्रति संवेदनशीलता। डिफ़ॉल्ट 1</p> है
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/orbitControl.mdx:152]: entire content should be wrapped under <p> tag

File: src/content/reference/hi/p5/lightFalloff.mdx [hunk 7]
```diff
@@ -1,35 +1,18 @@
 ---
 title: lightFalloff
 module: 3D
-submodule: Lights
+submodule: रोशनी
```
Comment (Divyansh013) [inline src/content/reference/hi/p5/lightFalloff.mdx:4]: use "प्रकाश" for lights since you have used that previously so it will be consistent

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

File: src/content/reference/hi/p5/lightFalloff.mdx (hunk 7 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/lightFalloff.mdx]: Done

File: src/content/reference/hi/p5/orbitControl.mdx (hunk 6 above)
Comment (jaiakash) [reply src/content/reference/hi/p5/orbitControl.mdx]: Done

**Review by Divyansh013 — APPROVED** [review]

**Review by TanviKumar — APPROVED** [review]

**Comment by Qianqianye** [comment]
Hi @jaiakash @Divyansh013 @TanviKumar, could you fix this PR so it could pass the checks? Thanks!

**Comment by jaiakash** [comment]
Hi @Qianqianye, the checks are passed now. Can you please merge now.
Thanks

**Comment by Qianqianye** [comment]
Looks great, thank you all!

**Pull request merged**